stream/stream.c

```diff
@@ -248,10 +248,10 @@ static int open_internal(const stream_info_t *sinfo, const char *url, int flags,
         s->access_references = opt;
     }
 
-    MP_VERBOSE(s, "Opening %s\n", url);
+    MP_DBG(s, "Opening %s\n", url);
```

Comment:
Not sure if I agree about this one?

Reply:
I agree with @wm4. This lets you know *how* a url is being opened via the msg module, which the user may find interesting.